src/components/organisms/Card/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { FC } from 'react';
+
+import CardActions from './CardActions/CardActions';
+import CardComponent, { CardProps } from './Card/Card';
+import CardContent from './CardContent/CardContent';
+import CardHeading from './CardHeading/CardHeading';
+import CardImage from './CardImage/CardImage';
+import CardText from './CardText/CardText';
+
+type CardStatic = {
+  Actions: typeof CardActions;
+  Content: typeof CardContent;
+  Heading: typeof CardHeading;
+  Image: typeof CardImage;
+  Text: typeof CardText;
+};
```

Comment:
Minor but probably TS is inferring this so no need to do it 💅🏻

Reply:
Otherwise, I can't tack other components onto `<Card />`